README.md

```diff
@@ -2,128 +2,14 @@
 
 Boost is a tool for Filecoin storage providers to manage data storage and retrievals on Filecoin.
 
-It replaces the `lotus-miner` `markets` service, as well as the `go-fil-markets` package with a standalone binary that runs alongside a `lotus` daemon and `lotus-miner`.
+See [Documentation](https://boost.filecoin.io/) to get started.
 
-Boost exposes `libp2p` interfaces for making storage and retrieval deals, a web interface for managing storage deals, and a GraphQL interface for accessing and updating real-time deal information.
+## For development:
 
-## Building
 
-### Prerequisites
+1. Install using instructions in [Documentation](https://boost.filecoin.io/).
 
-Currently, Boost installation is supported on macOS and Linux platforms.
-
-You need to have Go and Rust installed on your machine in order to build Boost, as well as some additional system dependencies, usually provided by your distribution: 
-
-Ubuntu/Debian:
-```
-sudo apt install mesa-opencl-icd ocl-icd-opencl-dev gcc git bzr jq pkg-config curl clang build-essential hwloc libhwloc-dev wget -y && sudo apt upgrade -y
-```
-
-Arch:
-```
-sudo pacman -Syu opencl-icd-loader gcc git bzr jq pkg-config opencl-icd-loader opencl-headers opencl-nvidia hwloc
-```
-
-macOS:
-```
-brew install go bzr jq pkg-config rustup hwloc coreutils
-```
```

Comment:
I don't think we're currently covering these on the docs site. If folks have already installed Lotus they shouldn't hit these problems, but I think we'll need to update on the docs site before deleting these sections.

Reply:
@LexLuthr , will you please update the docs to include this or let me know how I can? thanks

Reply:
Done

Reply:
thanks!  This should be ready to merge.